### PR TITLE
Create Travis autoupload guide

### PR DESCRIPTION
**Pull request by bfred-it**

### CODE REVIEW

File: Travis autoupload guide.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Travis autoupload guide
+
+With this setup, you can automatically deploy your extension when creating a tag.
+
+## Setup
+
+0. Enable your repo on travis
+0. Create a `.travis.yml` file and set your `EXTENSION_ID`:
+
+    ```yaml
```
Comment (zeke) [inline Travis autoupload guide.md]: why the indentation here?

Comment (bfred-it) [inline Travis autoupload guide.md]: That keeps it inside the `<li>`, otherwise it breakers the `<ul>`

Comment (zeke) [inline Travis autoupload guide.md]: 🙊

Comment (bfred-it) [inline Travis autoupload guide.md]: Good thing you noticed, though, because the other one is not indented and it looks weird now. Fixing...

**Comment by DrewML** [comment]
Appreciate all the PRs! This should help a lot.

**Pull request merged**